SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialReplicationGraph.cpp

```diff
@@ -90,6 +139,14 @@ void USpatialReplicationGraph::PostReplicateActors(UNetReplicationGraphConnectio
 	{
 		LoadBalancingHandler->ProcessMigrations();
 	}
+
+	USpatialNetDriver* SpatialNetDriver = Cast<USpatialNetDriver>(NetDriver);
+	if (bStrategyWorkerEnabled)
+	{
+		FSpatialLoadBalancingHandler::UpdateActorsHandedOver(*SpatialNetDriver, EntitiesHandedOver, ActorsHandedOver);
```

Comment:
Should `UpdateActorsHandedOver` do the emptying rather than doing it at this level?

Reply:
The Replication Graph populates them, so I think makes sense it depopulates them also.